etc/db_schema.xml

```diff
@@ -10,4 +10,7 @@
     <table name="sales_order">
         <column name="ga_user_id" xsi:type="varchar" comment="Google Analytics User ID for Server Side Analytics" nullable="true" length="255"/>
     </table>
+    <table name="quote">
```

Comment:
Please don't add columns to the `sales_quote` table as this table has the habit to contain a lot of rows. If you add this, all these rows need to be altered by the setup script, which can take a very very long time for some shops. I suggest creating a custom `sales_quote_ga` table or anything like that and add logic for that table in the module.